src/plugins/dashboard/public/dashboard_container/embeddable/dashboard_container.tsx

```diff
@@ -95,6 +95,8 @@ export class DashboardContainer extends Container<InheritedChildInput, Dashboard
   public subscriptions: Subscription = new Subscription();
   public controlGroup?: ControlGroupContainer;
 
+  public searchSessionId?: string;
```

Comment:
Want to add a comment about why this is in instance state instead of input state? I think that will be useful. Maybe something like

"searchSessionId is derived from input state. Storing as instance state instead of input state to avoid triggering input updates on input state changes"